translate/lang/data.py

```diff
@@ -239,6 +239,75 @@
 }
 
 
+scripts = {
+    'devanagari': [
+        'anp',
+        'bho',
+        'brx',
+        'doi',
+        'hi',
+        'kfy',
+        'kok',
+        'mai',
+        'mr',
+        'sa',
+        'sat',
+    ],
+    'bengali': [
+        'bn_BD',
+        'bn_IN',
```

Comment:
Since we simplify in script detection, can't we just specify `bn`.  The distinction in Mozillaland has more to do with politics than actual differences.